docker/entrypoint.sh

```diff
@@ -3,14 +3,11 @@
 ROOT_DIR=/usr/share/nginx/html
 
 # Replace env vars in JavaScript files
-api_url=$(echo "$VITE_API_URL" | sed 's/\//\\\//g') 
-for file in $ROOT_DIR/assets/*.js* $ROOT_DIR/index.html;
-do
-  echo "Processing $file ...";
+api_url=$(echo "$VITE_API_URL" | sed 's/\//\\\//g')
+for file in $ROOT_DIR/assets/*.js* $ROOT_DIR/index.html; do
+  echo "Processing $file ..."
 
   sed -i "s/VITE_API_URL_value/$api_url/g" $file 
-  sed -i 's|VITE_TOGGLE_value|'${VITE_TOGGLE}'|g' $file
-  sed -i 's|VITE_AUGUSTIN_COVER_value|'${VITE_TOGGLE}'|g' $file
-  sed -i 's|VITE_AGB_URL_value|'${VITE_AGB_URL}'|g' $file
+  sed -i 's|VITE_FRONTEND_URL_value|'${VITE_FRONTEND_URL}'|g' $file
```

Comment:
Why do we not send these two values from the backend as well?
I assume this is read from a specific file only used for the start up once?